src/install.rs

```diff
@@ -241,6 +241,20 @@ pub fn install(config: &InstallConfig) -> Result<()> {
         bail!("install failed");
     }
 
+    if let Ok(pts) = get_filesystems_with_label("boot") {
+        if pts.len() != 1 {
```

Comment:
```suggestion
        if pts.len() > 1 {
```

That way we're not going to print confusing messages if somehow we can't find the boot partition on the root device we just wrote.

Reply:
I guess to be even more precise, we can filter out from `pts` those that are on the root device, and then do `if !pts.is_empty() {`.

Reply:
ok, my intention was to warn if there's not just 1 `boot` pt